routes/api.php

```diff
@@ -1,8 +1,31 @@
 <?php
 
+use dnj\ErrorTracker\Laravel\Server\Http\Controllers\AppController;
+use dnj\ErrorTracker\Laravel\Server\Http\Controllers\DeviceController;
+use dnj\ErrorTracker\Laravel\Server\Http\Controllers\LogController;
 use Illuminate\Routing\Middleware\SubstituteBindings;
 use Illuminate\Support\Facades\Route;
 
 Route::middleware(['api', SubstituteBindings::class])->group(function () {
-    // TODO
+    Route::group(['prefix' => 'app', 'as' => 'app.'], function () {
```

Comment:
[ApiResources](https://laravel.com/docs/9.x/controllers#api-resource-routes) are not better?